fenix/app/src/main/res/values/dimens.xml

```diff
@@ -190,4 +190,8 @@
     <dimen name="accessibility_min_height">48dp</dimen>
 
     <dimen name="tap_increase_16">16dp</dimen>
+
+    <!-- Snackbar -->
+    <dimen tools:override="true" name="design_snackbar_min_width" tools:ignore="UnusedResources">-1px</dimen>
+    <dimen tools:override="true" name="design_snackbar_max_width" tools:ignore="UnusedResources">-1px</dimen>
```

Comment:
I don't have much knowledge here, but since the linter already flags these value as unused. I am wondering if we even need them?

Reply:
we could just add `tools:ignore="MissingDefaultResource"` to the s600dp folder to accommodate for not having values in the base folder. 

actually it might be the better option, because this code essentially is overriding the base values from the library by their duplicate. And the base values in the library (in a very unlikely event) might change.